modules/ob-creating-a-s2i-build.adoc

```diff
@@ -101,9 +102,9 @@ EOF
 .Example: Using `shp` CLI
 [source,terminal]
 ----
-$ shp build run s2i-nodejs-build --follow <1>
+$ shp build run s2i-nodejs-build --follow
 ----
-<1> Optional: By using the `--follow` flag, you can view the build logs in the output result.
+By using the `--follow` flag, you can view the build logs in the output result.
 
 . Check if the `BuildRun` resource is created by using one of the CLIs:
```

Comment:
This is out-of-scope for this review, but it is worth fixing in the future. When you do fix this, it's best to sweep your documentation set and fix all the places where this is used.

```suggestion
. Check if the `BuildRun` resource is created by running one of the following commands:
```